.github/workflows/benchmark.yml

```
@@ -78,3 +78,5 @@ jobs:
         comment-on-alert: true
         fail-on-alert: false
         alert-comment-cc-users: '@giovannipizzi,@agoscinski,@GeigerJ2,@khsrali,@unkcpz'
+      # if a pull request is merged from the upstream
+      if: github.event_name != 'pull_request' || not github.event.pull_request.head.repo.fork
```

I think we should only publish when this is merged to main. It does not make sense to me to publish a benchmark result in an open PR. What do you think @danielhollas ?
```suggestion
      if: github.event_name == 'push' && github.ref_name == 'main'
```

---

I changed it